get-image-info.py

```diff
@@ -75,6 +75,10 @@ def main(args, start_time):
 								len(app_list), num_bad,
 								num_xtrnl, num_ntrnl])
 
+def send_results():
```

Comment:
Why are there two functions with the exact same name? And it looks like they are never called...